source/material_model/grain_size.cc

```diff
@@ -211,6 +211,51 @@ namespace aspect
 
 
 
+    template <int dim>
+    double
+    GrainSize<dim>::
+    moment_of_grain_size_distribution (const int n) const
+    {
+      // This function normalizes the grain size distribution using the nth moment.
+      // Description can be found in eq 8 of Bercovici and Ricard (2012)
+      const double sigma = 0.8;
+
+      return exp (n * n * sigma * sigma / 2.);
+    }
+
+
+    template <int dim>
+    double
+    GrainSize<dim>::
+    roughness_to_grain_size_factor (const double volume_fraction_phase_one)  const
+    {
+      // This factor is used to convert the roughness equation into mean grain size
+      // Refer to Appendix H.1, eqs 8, F.28 in Bercovici and Ricard (2012) for more details.
+      const double b1 = 1./20 ;
+      const double c1 = 3 * b1 * moment_of_grain_size_distribution(4) / (8 * moment_of_grain_size_distribution (2));
```

Comment:
```suggestion
      const double c1 = 3.0 * b1 * moment_of_grain_size_distribution(4) / (8.0 * moment_of_grain_size_distribution (2));
```

Also, if this is the only place where it is used, I wonder if it is really worth making it a  separate function. We can discuss in our meeting today.